audb/core/publish.py

```diff
@@ -447,17 +538,16 @@ def publish(
                 verbose)
 
     # publish media
-    media = _find_media(db, db_root, version, deps, archives, num_workers,
-                        verbose)
-    _put_media(media, db_root, db.name, version, deps, backend, num_workers,
-               verbose)
+    media = _find_media(db, db_root, db_root_files, version, deps, archives,
+                        num_workers, verbose)
```

Comment:
The `_find_media()` function is not so easy to understand as it seems to do two different things:
* updating `deps` with information over (new?) media files
* collecting a list of archives that is returned (but strangely named `media` here)

Maybe you can add a docstring to the function and describe what it does?
And maybe rename `media` here to `media_archives` to be more precise, but do not collide with the `archives` argument.

Reply:
Great, thanks for splitting the two function in a better way. Now it is indeed easier to understand.